state-chain/chains/src/sol.rs

```diff
@@ -191,11 +191,14 @@ impl FeeEstimationApi<Solana> for SolTrackedData {
 				},
 		);
 
+		// Match the minimum broadcast prio fee
+		let priority_fee = sp_std::cmp::max(self.priority_fee, MIN_COMPUTE_PRICE);
```

Comment:
we are maxing a price and against fee? these are two different things no?

Reply:
Namings are not great.. I think we named Solana's "compute_units" and "compute_price" as "gas" and "priority_fee" after the EVM so it'd be easy to follow in our codebase, this way it was the same concepts across chains. But then it ends up with  some things like this that don't look right.
Priority fees is the same as compute price.

Reply:
To be fair, Solana's rpc calls are for "priorization fees" but then when setting them into an instruction is called "compute_price". So their naming is not consistent.

Reply:
I think just different words for the same thing (depending on your pov, if you're charging or paying). The price is the minimum fee you need to pay.

Reply:
yeah it seems like a naming thing - we could at the very least clarify with a doc comment that it is an absolute value. when I see "price" I'm thinking ratio of two things, but that's not the case here

Reply:
yes. The compute price is the literal price of a compute unit, to me it makes more sense than calling it priority fee. I'll just clarify it in a comment.